docker/Dockerfile

```diff
@@ -17,3 +19,15 @@ RUN install2.r --error remotes &&  \
 
 COPY . /src
 RUN R CMD INSTALL --install-tests /src && rm -rf /src
+
+ENV ORDERLY_RUNNER_QUEUE_ID=orderly.runner
+ENV REDIS_CONTAINER_NAME=orderly.runner_redis
```

Comment:
packit does not need a redis container, after addressing rich's comments, this will come from env args in docker run and so these variables will come from the packit deploy tool which makes sense